src/components/box/Box.jsx

```diff
@@ -26,7 +26,7 @@ export const PADDING = {
 };
 
 const Box = ({color, padding, full, children, border = !color, imgSrc, noMinHeight, shadow, noBorderRadius,
-  onClose, className, ...props}) => {
+  onClose, whiteCloseIcon, className, ...props}) => {
```

Comment:
I don't think we should be that much specific, can we just use closeIconColor with allowed light & dark?

Reply:
and if not, I would suggest to use `light` instead